pkg/p2p/impl.go

```diff
@@ -280,7 +279,12 @@ func (n *p2pNetwork) connectToRemoteNodes() {
 			go func() {
 				for {
 					if n.shouldConnectTo(remoteNode.NodeInfo) {
-						if peer, err := remoteNode.connect(n.config); err != nil {
+						tlsConfig := tls.Config{
+							Certificates:       []tls.Certificate{n.config.ClientCert},
+							InsecureSkipVerify: true, // TODO: Is the actually secure?
```

Comment:
Is there an issue for this TODO? Looks important

Reply:
Made an issue for it.

https://github.com/nuts-foundation/nuts-network/issues/15